docs/FeatureManagement/FeatureManagement.v1.0.0.schema.json

```diff
@@ -23,7 +23,7 @@
                     "title": "Feature Flags",
                     "description": "Declares feature flags based on Microsoft Feature Flag schema.",
                     "items": {
-                        "$ref": "https://raw.githubusercontent.com/Azure/AppConfiguration/main/docs/FeatureManagement/FeatureFlag.v1.1.0.schema.json"
+                        "$ref": "https://raw.githubusercontent.com/Azure/AppConfiguration/main/docs/FeatureManagement/FeatureFlag.v2.0.0.schema.json"
```

Comment:
I'm not sure this is the right way to do this. Should we create a FeatureManagement.**v2.0.0**.schema.json instead?

Reply:
I am ok with create FM v2 schema. If we have FF v2.1 schema in the future, should we just update the reference in FM v2 schema or create a new FM v2.1 schema?

Reply:
We shouldn't change the FF schema version in place, so it should be FM v2.1 schema.